code/platform_win32/4ed_dx11_render.cpp

```diff
@@ -560,6 +560,9 @@ gl_render(Render_Target *t){
          group = group->next){
         Rect_i32 box = Ri32(group->clip_box);
         
+        // NOTE(FS): Ignore this group if we our scissor rectangle is not well defined
+        if ((rect_width(box) <= 0) || (rect_height(box) <= 0)) continue;
```

Comment:
I would prefer to use this style for ifs, like the rest of the file.
```suggestion
        if ((rect_width(box) <= 0) || (rect_height(box) <= 0)) {
            continue;
        }
```